core/templates/pages/about-foundation-page/about-foundation-page.component.html

```diff
@@ -11,35 +11,19 @@
                class="oppia-about-foundation-page-oppia-logo"
                src="/assets/images/about_foundation/oppia_big_logo.png">
         </picture>
-        <h1>
-          The Oppia Foundation
-        </h1>
-        <h3>
-          Building a community to provide quality education to those who lack access to it
-        </h3>
+        <h1 [innerHTML]="'I18N_ABOUT_FOUNDATION_PAGE_HEADING' | translate"></h1>
+        <h3 [innerHTML]="'I18N_ABOUT_FOUNDATION_PAGE_HEADING_CONTENT' | translate"></h3>
```

Comment:
For variables that don't point to a html snippet, prefer using data interpolation instead of innerHTML. Please change all instances of this.
```suggestion
        <h1>{{ 'I18N_ABOUT_FOUNDATION_PAGE_HEADING' | translate }}</h1>
        <h3>{{ 'I18N_ABOUT_FOUNDATION_PAGE_HEADING_CONTENT' | translate }}</h3>
```

Reply:
Ok, i made the changes as you requested.
But can i ask if it makes any difference in performance if we're using either?